src/include/rbd/librbd.h

```diff
@@ -301,7 +301,8 @@ typedef enum {
 } rbd_group_snap_state_t;
 
 typedef enum {
-  RBD_GROUP_SNAP_NAMESPACE_TYPE_USER = 0
+  RBD_GROUP_SNAP_NAMESPACE_TYPE_USER = 0,
+  RBD_GROUP_SNAP_NAMESPACE_TYPE_MIRROR
```

Comment:
Nit: RBD_GROUP_SNAP_NAMESPACE_TYPE_MIRROR = 1

Reply:
Isn't it redundant?